app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new(text: comment_params[:text], author_id: current_user.id, post_id: @post.id)
+
+    if @comment.save
+      redirect_to "/users/#{@post.author_id}/posts/#{@post.id}", notice: 'Success!'
+    else
+      render :new, alert: 'Error occured!'
+    end
+  end
```

Comment:
Please ensure to use flash messages for successful and not successful paths in your `comment create action` according to the project requirement.

Reply:
- [ ] Like the last code reviewer said you should add a flash message for comments. You already did in like and post controller. Please add it also the comment controller. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec_n%2B1.md#:~:text=Add%20flash%20messages%20in%20the%20create%20actions%20in%20all%20your%20controllers.)